go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/codeready-toolchain/host-operator
 
 require (
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
```

Comment:
why do we suddenly need that `assert` pkg? 🤔

Reply:
don't ask me - this is automatically pulled when I run `go mod tidy` on master without any changes.
btw, it's indirect dependency.

Reply:
most likely transitive dependency of 
https://github.com/codeready-toolchain/host-operator/commit/d72e68a0c78ab9a1cb14c80d488f5628f2a61d51#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R17
as this is the commit that introduced it